tests/test_readme.py

```diff
@@ -50,7 +51,7 @@ def extract_code_snippets() -> tuple[dict[str, str], list[str]]:
               `test:`) to the code string.
             - **snippets_unnamed**: list of code strings from unnamed test blocks.
     """
-    readme_content = Path("README.md").read_text(encoding="utf-8")
+    readme_content = (repo_root / "README.md").read_text(encoding="utf-8")
```

Comment:
... just use 


```suggestion
    readme_content = (pytestconfig.rootpath / "README.md").read_text(encoding="utf-8")
```

here? I think the fixture is a bit overkill although it works 😅